supervisor_docker.conf

```diff
@@ -1,14 +1,3 @@
-[unix_http_server]
-file=/tmp/supervisor.sock
-chmod=0700
-
-[supervisorctl]
-serverurl=unix:///tmp/supervisor.sock
-
-[rpcinterface:supervisor]
-supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
-
-
```

Comment:
This is just for debugging purpose to see the processes in the terminal, felt not needed.